portal/templates/initial_queries.html

```diff
@@ -11,12 +11,12 @@
       {{tou(terms)}}
   {% endif %}
   <div id="aboutForm" class="tnth-hide">
+    {% if user and user._email %}
```

Comment:
@achen2401 anything with an underscore in front of it, by convention, is considered a hidden (similar to private) variable in python.  There are exceptions, but you don't typically want to access these outside of the class.

Is there a reason we're looking at `user._email` here rather than `user.email`?  The hidden value holds the INVITE and NO_EMAIL prefixes, which I don't think should ever be seen or used at the UI level.